controllers/controllers/services/bindings/controller.go

```diff
@@ -145,6 +144,15 @@ func (r *Reconciler) ReconcileResource(ctx context.Context, cfServiceBinding *ko
 		return res, err
 	}
 
+	if cfServiceBinding.Labels[korifiv1alpha1.ServiceCredentialBindingTypeLabel] == korifiv1alpha1.CFServiceBindingTypeApp {
```

Comment:
Why do we do this? The old version does not lookup the app and the new version should be concerned with bindings of type `key`.

Reply:
When the type of the binding is "key" there is no app associated to it and the call to the k8s api returns an error two lines below the if statement: 
		`err = r.k8sClient.Get(ctx, types.NamespacedName{Name: cfServiceBinding.Spec.AppRef.Name, Namespace: cfServiceBinding.Namespace}, cfApp)`
Do we need to remove the app fetching logic entirely?